.github/workflows/test.yml

```diff
@@ -2,9 +2,9 @@ name: Lint and Test
 
 on:
   pull_request:
-    branches: [main]
+    branches: [main,v1.9.0]
```

Comment:
Branch `v1.9.0` doesn't exist in this repo. Please omit the change to this file from the PR.